src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+    private final String eventFrom;
+    private final String eventTo;
```

Comment:
Consider renaming these constant variables to a name in all uppercase, in adherence to the module's coding standard.